checks.d/mysql.py

```diff
@@ -330,9 +330,10 @@ def _get_config(self, instance):
         options = instance.get('options', {})
         queries = instance.get('queries', [])
         ssl = instance.get('ssl', {})
+        connect_timeout = instance.get('connect_timeout', None)
```

Comment:
I was wondering if we had to cast the `instance.get('connect_timeout')` to an int when it wasn't `None`, but it doesn't seem to be needed.

We have to keep the default to `None` for backward compatibility, even if it would probably better to set one at 5 seconds or so. Thanks for the details!